resources/analysisTools/indelCallingWorkflow/checkSampleSwap_TiN.pl

```diff
@@ -315,10 +321,12 @@
 
 close GermlineRareFile;
 close SomaticFile;
-close Ann;
+close ANN;
```

Comment:
Seems to have been a typo missed before, because of the missing `use warnings;`.